README.md

```diff
@@ -35,3 +35,31 @@ In a separate window, compile the project and run the migrations, deploying the
 `truffle compile`
 
 `truffle migrate`
+
+### Deploying to Rinkeby test network
+
+First things first you need a Rinkeby account with some Rinkeby ether in it. The [official Ethereum Wallet](https://ethereum.org/) is a good place to start.
+
+You can get Rinkeby ether from [the Rinkeby faucet](https://www.rinkeby.io/#faucet).
+
+We need to tell Rinkeby and Truffle the address of this deploying account. We have therefore introduced a  CREATOR_ACCOUNT environment variable that returns the address of the deploying account.
+
+You can set this manually or install direnv (`brew install direnv`) and update .envrc with the address of your account.
+
+Easiest way to run a Rinkeby node locally is using geth:
+
+`brew tap ethereum/ethereum`
+
+`brew install ethereum`
+
+You can then start it as follows:
+
+`geth --rinkeby --rpc --unlock="$CREATOR_ACCOUNT"`
```

Comment:
You'll need to close you Mist Ethereum Wallet in order to run this command

Reply:
I discovered you can run them both at the same time by starting Mist from the terminal and passing it the path to rinkeby's IPC file, which is printed out by geth after you start the network. So the command for me was: `/Applications/Mist.app/Contents/MacOS/Mist --rpc /Users/aliblackwell/Library/Ethereum/rinkeby/geth.ipc`